openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -459,3 +461,28 @@ def _delete_cookie(response):
         secure=settings.SESSION_COOKIE_SECURE or None,
         httponly=settings.SESSION_COOKIE_HTTPONLY or None,
     )
+
+
+def _is_from_logout(request):
+    """
+    Returns whether the request has come from logout action to see if
+    'is_from_logout' attribute is present.
+    """
+    return getattr(request, 'is_from_logout', False)
+
+
+@contextmanager
+def controlled_logging(request, logger):
+    """
+    Control the logging by changing logger's level if
+    the request is from logout.
+    """
+    if _is_from_logout(request):
+        logger.setLevel(ERROR)
+
+    try:
+        yield
+
+    finally:
+        if logger.getEffectiveLevel() == ERROR:
+            logger.setLevel(INFO)
```

Comment:
If that's not feasible we can store the original logging level after getting logger through `getLogger(__name__)` to restore back later?